### ISSUE TITLE
$('<tr>') produces empty array

### ISSUE THREAD

**Issue by limonte** [kenwheeler/cash#204]
```js
$.noConflict()

console.log($('<tr>')[0])        // undefined
console.log(jQuery('<tr>')[0])   // tr element, as expected
```

On the other hand, this works:

```js
console.log($(document.createElement('tr'))[0]) // tr element, as expected
```

live demo: https://jsfiddle.net/6L4fep8s/5/

**Comment by fabiospampinato** [comment]
We are parsing html strings by inserting them into the body of a new document, and I think a `<tr>` element outside of a `<table>` isn't valid html, that's why it's not parsed as you might expect. I think jQuery is doing some extra checks for these edge cases. 

It's probably not worth it doing the same as jQuery here, I guess there's probably a considerable amount of code involved.

As a workaround you can do this:

```js
$('<table><tr></tr></table>').find ( 'tr' )[0] // tr element
```

**Comment by fabiospampinato** [comment]
Fixed in v2.3.8.
